streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -3494,32 +3323,27 @@ private StreamThread setUpThread(final Properties streamsConfigProps) {
     }
 
     private TaskManager mockTaskManager(final Task runningTask) {
-        final TaskManager taskManager = EasyMock.createNiceMock(TaskManager.class);
+        final TaskManager taskManager = mock(TaskManager.class);
         final TaskId taskId = new TaskId(0, 0);
 
-        expect(runningTask.state()).andStubReturn(Task.State.RUNNING);
-        expect(runningTask.id()).andStubReturn(taskId);
-        expect(taskManager.allOwnedTasks()).andStubReturn(Collections.singletonMap(taskId, runningTask));
-        expect(taskManager.commit(Collections.singleton(runningTask))).andStubReturn(1);
+        when(runningTask.state()).thenReturn(Task.State.RUNNING);
+        when(taskManager.allOwnedTasks()).thenReturn(Collections.singletonMap(taskId, runningTask));
         return taskManager;
     }
 
     private TaskManager mockTaskManagerPurge(final int numberOfPurges) {
         final Task runningTask = mock(Task.class);
         final TaskManager taskManager = mockTaskManager(runningTask);
 
-        taskManager.maybePurgeCommittedRecords();
-        EasyMock.expectLastCall().times(numberOfPurges);
```

Comment:
This needs to be transformed in a verification in Mockito.

Reply:
Yes, sorry, I don't know why I left/change it to this, I changed it everywhere else to what you have suggested. This has been remedied in the latest version of the pull request.